### PR TITLE
Remove only xCAT entries from rsyslog.conf file

### PR DESCRIPTION
**Pull request by gurevichmark**
Mark the start and end of the xCAT section in rsyslog.conf file. 
Then, when needed, only remove lines between those 2 markers.

### CODE REVIEW

File: xCAT/postscripts/syslog [hunk 1]
```diff
@@ -182,7 +187,8 @@ config_rsyslog_V8()
     # to prevent duplicate logs after "syslog" script is run 
     sed -i '/^[^#].*\/var\/log\/messages/s/^/#/' $conf_file
 
-    echo "# xCAT settings" >> $conf_file
+    # Mark the start of xCAT section
+    echo "# $xCATSettingsSTART" >> $conf_file
     echo "#*.debug   /var/log/messages"  >> $conf_file
```
Comment (whowutwut) [inline xCAT/postscripts/syslog]: @neo954  Do you recall why we echo out this statement commented out?

File: xCAT/postscripts/syslog [hunk 2]
```diff
@@ -182,7 +187,8 @@ config_rsyslog_V8()
     # to prevent duplicate logs after "syslog" script is run 
     sed -i '/^[^#].*\/var\/log\/messages/s/^/#/' $conf_file
 
-    echo "# xCAT settings" >> $conf_file
+    # Mark the start of xCAT section
+    echo "# $xCATSettingsSTART" >> $conf_file
```
Comment (whowutwut) [inline xCAT/postscripts/syslog]: Should we add a comment in here informing users that changes between the START and END tags will be overwritten by xCAT?

Comment (gurevichmark) [inline xCAT/postscripts/syslog]: Added text to let the user know about entries being replaced.

**Comment by whowutwut** [comment]
I'm ok with the changes

**Comment by gurevichmark** [comment]
@whowutwut @daniceexi Do you guys think this needs to be cherry picked into 2.11 ?

**Comment by whowutwut** [comment]
Cherry picked to 2.11 branch:
commit ccb5723c3bae4b7d43b5c9d28872eade806c050b
commit 031892a587266424aa4f780b5d9e9252c4eacb92

**Pull request merged**